lint.sh

```diff
@@ -6,7 +6,7 @@ function incr_err() {
 }
 
 # check that mock.patch calls use autospec
-OUTPUT=`git grep "patch(" | grep -v "autospec"`
+OUTPUT=`git grep "mock.patch(" | grep -v "autospec"`
```

Comment:
Can we support this either being `mock.patch` or `patch()` with no preceeding thing?

This no longer catches the case wherein you do:

```python
from mock import patch
patch(...)
```